stylelint-config/index.js

```diff
@@ -2,84 +2,24 @@
 
 module.exports = {
   "extends": [
-    "stylelint-config-recommended-scss",
+    "stylelint-config-standard-scss",
   ],
   plugins: [
     "stylelint-scss"
   ],
-  "rules": {
-    "color-hex-case": "lower",
-    "color-hex-length": "short",
-    "color-no-invalid-hex": true,
-
-    "number-leading-zero": "always",
+  rules: {
```

Comment:
Found this rules in React Todo App with API
```suggestion
  rules: {
    "alpha-value-notation": null,
    "color-function-notation": null,
    "no-invalid-position-at-import-rule": null,
    "property-no-vendor-prefix": null,
    "value-keyword-case": null,
```